Sprint-3/implement/get-card-value.js

```diff
@@ -31,41 +31,19 @@
 // Then it should throw an error indicating "Invalid card rank."
 
 function getCardValue(card) {
-  if (typeof card !== "string") return "Input should be a sting.";
-  const cardValue = card
-    .toUpperCase()
-    .substring(0, card.indexOf(card.slice(-1)));
+  if (typeof card !== "string") return "Input should be a string.";
+  const cardValue = card.slice(0, -1).toUpperCase();
+  const allCards = ["A", "K", "Q", "J", "10", "9", "8", "7", "6", "5", "4", "3", "2", "1"];
   const faceCards = ["K", "Q", "J"];
 
-  /* If we wanted to exlude cards for not being a suite
-   const suites = ["♣", "♠", "♦", "♥"];
-   if (!suites.includes(card.slice(-1))) return "Invalid card rank.";
-  */
-  if (Number(cardValue) && Number(cardValue) >= 2 && Number(cardValue) <= 10)
-    return Number(cardValue);
-  if (faceCards.includes(cardValue)) return 10;
-  if (cardValue === "A") return 11;
+  if (allCards.includes(cardValue)) {
+    if (Number(cardValue) && Number(cardValue) >= 2 && Number(cardValue) <= 10)
+      return Number(cardValue);
+    if (faceCards.includes(cardValue)) return 10;
+    if (cardValue === "A") return 11;
```

Comment:
There is a way to rewrite the code at line 40-43 without the if statement at line 40.

Reply:
Ok, I got you on that one. I can literally rewrite it without the if statement on line 40!

I've removed the if statement and moved the `return Number(cardValue)`  after line 43.